client/lib/wpcom-undocumented/lib/site.js

```diff
@@ -217,6 +217,17 @@ UndocumentedSite.prototype.newExport = function( fn ) {
 };
 
 /**
+ * Returns media storage limits and space used for a given site. If site has
+ * unlimited storage or is a jetpack site, values returned will be -1.
+ *
+ * @param {Function} callback - called on completion of the GET request
+ * @return {Object} promise - resolves on completion of the GET request
+ */
+UndocumentedSite.prototype.getMediaStorage = function( callback ) {
```

Comment:
The naming of this is inconsistent with typical WPCOM.js naming, where `get` is generally omitted (granted, there's several other inconsistencies within this same file alone).